ci/deps/travis-36-cov.yaml

```diff
@@ -15,6 +15,7 @@ dependencies:
   # pandas dependencies
   - beautifulsoup4
   - botocore>=1.11
+  - cftime
```

Comment:
Let me know if this is the correct CI environment to add a dependency for the downstream test I wrote.

Reply:
id add this to any of the environment files that already has xarray in it, along with a comment about what its needed for